### DISCUSSION TITLE
Jaeger/ Open Telemetry dropping child spans

### DISCUSSION THREAD

**Discussion by vishnusaikumar** [jaegertracing/jaeger#4335] [Q&A]
We have DockerFile which uses Jaeger all in one image

![image](https://user-images.githubusercontent.com/16996345/227166283-e904c977-1ce1-40a7-9f0b-734288f97bf0.png)

Now we have Kafka connect with OpenTelemetry Java agent configured as below

![image](https://user-images.githubusercontent.com/16996345/227167224-cce16607-e6a8-4d17-9dc6-ae38177cace3.png)

and at kafka consumer we have configured similarly

![image](https://user-images.githubusercontent.com/16996345/227168139-fd7f8dcf-c147-41b4-94da-39fd9b94e4fe.png)

but the number of spans at consumer is always less than at producer topic even though the consumption is happening properly

![image](https://user-images.githubusercontent.com/16996345/227168622-e9bfd066-54a4-42a3-b634-294e9c1b47e0.png)

similarly multiple child spans are getting dropped
we are using spring boot kafka listener

![image](https://user-images.githubusercontent.com/16996345/227169375-495b9c83-8f91-4f9f-9ab6-741b90cccbf8.png)

Is there any additional configuration that needs be added so that all traces will be picked at consumer correctly?

**Comment by vishnusaikumar** [comment]
We are using Spring Boot **2.3**
and
Spring Kafka **2.5.1** Release

**Comment by yurishkuro** [comment]
I don't see any reason to suspect that this is Jaeger's issue, rather than instrumentation or setup, so can't offer any suggestions.

**Reply by vishnusaikumar** [reply]
Thank you for getting back
I have raised a discussion thread at 
https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions/8126

I'm suspecting on our older version of Spring Kafka, since the latest changes of instrumentation might have resolved any such issues.
